app/controllers/requests_controller.rb

```diff
@@ -58,17 +58,11 @@ def create
   def update
     respond_to do |format|
       prepare_params
+      @request.assign_sudo_users(request_params[:sudo_user_ids][1..-1])
```

Comment:
Hm, I thought of something similar as well, the [1..-1] here removes the first element, because for some reason, the parameter always contains one empty string at the beginning.
but we could actually move that into the prepare_params code, I think that would make sense.